components/docs-chef-io/content/automate/chef_infra_server.md

```diff
@@ -81,14 +83,54 @@ To access these components for an organization, select **Chef Servers > Organiza
 
 A [cookbook]({{< relref "cookbooks" >}}) is the fundamental unit of configuration and policy distribution. A cookbook contains recipes and other files, components, or directories.
 
+The Chef Infra Server lets you view:
+
+- List of cookbooks with their latest version
+- Different versions of a cookbook
+- Contents of a cookbook
+- Details of a cookbook
+
+#### List of Cookbooks
+
+In Chef Infra Server, you can view all the cookbooks of an organization. The latest version of the cookbooks are mentioned in the list.
+
+{{< figure src="/images/automate/list-of-cookbooks.png" alt="List of Cookbooks">}}
+
+Select a cookbook from the list to view the details of that cookbook.
+
+#### Different Versions of Cookbooks
+
+To view different versions of cookbook:
+
+- Select a cookbook from the list.
+- The interface contains a dropdown list for versions. Select any version to view the details.
+
+{{< note >}}
+The dropdown shows the latest version of the cookbook.
+{{< /note >}}
+
+{{< figure src="/images/automate/view-different-versions-of-cookbook.png" alt="Versions of a Cookbook">}}
+
+#### Cookbook Content
+
+Select the content tab to view the recipes and other optional components of a cookbook. It also contains the files or directories of the components. To view the cookbook content, select the file from the component list.
+
+{{< figure src="/images/automate/contents-of-a-cookbook.png" alt="Cookbook Contents">}}
+
+#### Details of a Cookbook
+
+Select the **Details** tab to view the requirements, usage, resources, and license of a cookbook.
+
+{{< figure src="/images/automate/details-of-a-cookbook.png" alt="Details of a Cookbook">}}
+
 ### Roles
 
 [Roles]({{< relref "roles" >}}) let you define patterns and processes that exist across nodes in an organization as belonging to a single job function. Each role consists of zero (or more) attributes and a run-list.
 
-The Chef Infra Server UI lets you:
+The Chef Infra Server lets you:
 
 - Create a role
-- List out all the roles at one place.
+- List all the roles at one place.
```

Comment:
This is still wordy. It also attributes listing to the user, even though the server creates the list. Consider something like `View all roles` or `View all roles in an organization` and `Search for a specific role`.